vulnfeeds/cmd/combine-to-osv/main.go

```diff
@@ -159,9 +160,20 @@ func combineIntoOSV(loadedCves map[cves.CVEID]cves.Vulnerability, allParts map[c
 				convertedCve.Withdrawn = modified
 			}
 		}
+
+		addedDebianURL := false
+		addedAlpineURL := false
 		for _, pkgInfo := range allParts[cveId] {
 			convertedCve.AddPkgInfo(pkgInfo)
+			if strings.HasPrefix(pkgInfo.Ecosystem, "Debian") && !addedDebianURL {
```

Comment:
Set these "Debian" and "Alpine" strings to be constants. This way we can get type checking in the if statements in addReference as well.